packages/kit/test/apps/basics/package.json

```
@@ -6,8 +6,7 @@
 		"dev": "vite dev",
 		"build": "vite build",
 		"preview": "vite preview",
-		"prepare": "node ../../cli.js sync",
-		"check": "tsc && svelte-check",
+		"check": "svelte-kit sync && tsc && svelte-check",
```

```suggestion
		"check": "node ../../cli.js sync && tsc && svelte-check",
```
without this is always goes to dist for me.  
And do we want to keep the prepare script too to work around the fact that pnpm doesn't call postinstall for workspace dependencies in a monorepo?

---

Ah yeah, changed it to `../../cli.js` so we can test out changes to the `sync` script without rebuilding.

IIRC the `prepare` script was running before stuff was built, which meant it didn't work (though that would have been fixed by the `../../cli.js` I guess) but in any case `"prepare": "svelte-kit sync"` is now a no-op, so there's no reason to keep it

---

I was thinking of prepare with the ../../cli.js call too, so that when you pull changes and do `pnpm i` it syncs to avoid red squiggles.